app/modules/dev/incident.py

```diff
@@ -0,0 +1,36 @@
+import json
```

Comment:
I like how you are splitting the incidents logic for dev and "real" incidents.

Reply:
Trying to make it easier for folks to jump on the project and collaborate down the road!

One of the next iteration will more gracefully handle the "dev incident" records in a local DB so that we don't have to parse the the details from the actual DB and yet still test incident related features in incident-dev- channels